game/templates/game/board.html

```
@@ -537,40 +411,84 @@
+            <div id="streak-counter" class="streak-counter" style="display:none;">
+                🔥 <span id="streak-count">0</span> Day Streak
             </div>
         </div>
-        
-<!-- Right Panel: Controls -->
-        <div class="panel">
-            <div class="card">
+
+        <!-- Consolidated Sidebar -->
+        <div class="sidebar-container" id="mobilePanel">
+            <button id="mobileControlsClose" aria-label="Close controls" class="mobile-close-btn">&times;</button>
```

HTMLHint (1.9.2)

[warning] The type attribute must be present on <button> elements.

(button-type-require)